deploy/terraform-aws/cleanup-vpc.sh

```diff
@@ -55,9 +55,16 @@ get_pending_and_running_instance_ids_by_vpc_id() {
 instance_ids=$(get_pending_and_running_instance_ids_by_vpc_id "$VPC_ID")
 while [[ -n "$instance_ids" ]]
 do
-    echo "Terminating instances: $instance_ids"
-    aws ec2 terminate-instances --instance-ids "$instance_ids"
-    aws ec2 wait instance-terminated --instance-ids "$instance_ids"
+    for instance_id in $instance_ids
+    do
+        echo "Terminating instance: $instance_id"
+        aws ec2 terminate-instances --instance-ids "$instance_id"
+    done
+    for instance_id in $instance_ids
+    do
+        aws ec2 wait instance-terminated --instance-ids "$instance_id"
```

Comment:
does wait halt execution until it sees that the id is in a state of terminate or does this run async and poll until the state becomes terminated?